lib/main_common.pm

```diff
@@ -1669,8 +1669,19 @@ sub load_wicked_tests {
         loadtest 'wicked/advanced/' . $f . '/t12_bridge_interface_xml';
     }
     elsif (check_var('WICKED', 'startandstop')) {
-        loadtest 'wicked/startandstop_ref' if check_var('IS_WICKED_REF', '1');
-        loadtest 'wicked/startandstop_sut' if check_var('IS_WICKED_REF', '0');
+        my $f = get_var('IS_WICKED_REF') ? 'ref' : 'sut';
+        loadtest 'wicked/startandstop/' . $f . '/t01_standalone_card_ifdown_ifreload';
+        loadtest 'wicked/startandstop/' . $f . '/t02_bridge_ifreload';
+        loadtest 'wicked/startandstop/' . $f . '/t03_bridge_ifup_ifreload';
+        loadtest 'wicked/startandstop/' . $f . '/t04_bridge_ifup_remove_all_config_ifreload';
+        loadtest 'wicked/startandstop/' . $f . '/t05_bridge_ifup_remove_one_config_ifreload';
+        loadtest 'wicked/startandstop/' . $f . '/t06_bridge_ifdown_create_new_config_ifreload_ifdown_ifup';
+        loadtest 'wicked/startandstop/' . $f . '/t07_bridge_ifdown_remove_one_config_ifreload_ifdown_ifup';
+        #loadtest 'wicked/startandstop/' . $f . '/t08_vlan_ifdown_modify_config_ifreload';
+        #loadtest 'wicked/startandstop/' . $f . '/t09_vlan_ifdown_modify_one_config_ifreload_ifdown_ifup';
+        #loadtest 'wicked/startandstop/' . $f . '/t10_vlan_ifup_all_ifdown_one_card';
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

Reply:
sure, it can always be enabled later on